distribution/sender/routes/shutdown-route.xml

```diff
@@ -6,8 +6,14 @@
             <method beanType="org.openmrs.eip.app.CustomFileOffsetBackingStore" method="disable()" />
         </script>
         
-        <!-- TODO Send a notification to the configured emails -->
-
+        <doTry>
+        	<toD uri="direct:send-notification-email-route" />
```

Comment:
Call the processor directly from here and pass it the sender from here on the exchange